release-docker.sh

```diff
@@ -1,14 +1,25 @@
-#!/bin/bash
+#!/bin/sh
 
-tag=$(git describe --exact-match 2>/dev/null)
-if [[ $? != 0 ]]; then
-	tag=latest
+set -eu
+
+if ! tag="$(git describe --exact-match 2> "/dev/null")"; then
+	tag="latest"
 fi
 
-docker build -t mvdan/shfmt:$tag -f cmd/shfmt/Dockerfile .
+docker build --tag "mvdan/shfmt:${tag}" \
+             --file "cmd/shfmt/Dockerfile" \
+             "./"
+
+docker build --tag "mvdan/shfmt:${tag}-alpine" \
+             --target "alpine" \
+             --file "cmd/shfmt/Dockerfile" \
+             "."
```

Comment:
follow `shfmt` :)

Reply:
I'll reply, even though the file has been deleted now ;) Just because :p
You are right, I didn't, I had forgotten and my OCD wanted to make it look pretty. I'll try to remember to use your style here :)